bin/boot

```diff
@@ -25,6 +25,36 @@ function login  {
 }
 
 function start {
+    if [ "$ADMIN_PASSWORD" ]; then
+        sed -i "s/_ADMIN_PASSWORD_/$ADMIN_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_USER" ]; then
+        sed -i "s/_DB_USER_/$DB_USER/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_PASSWORD" ]; then
+        sed -i "s/_DB_PASSWORD_/$DB_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$ADDONS_REPO" ]; then
+        echo "Setting up SSH key"
+        cp -R /mnt/ssh /root/.ssh
+        chown -R $USER:$USER /root/.ssh
+        ssh_file="/root/.ssh/id_rsa"
+        if [ -f "$ssh_file" ]; then
+            chmod 600 /root/.ssh/id_rsa
+        fi
+        ssh-keyscan github.com  >> /root/.ssh/known_hosts
```

Comment:
What if we use other git hosting services, e.g. BitBucket or private GitLab? Ideally, the keyscan should happen on every host